app/views/home/businesses.html.erb

```diff
@@ -1,128 +1,119 @@
-            <footer class="alt-text-small">Christine Abernathy (<a href="https://github.com/caabernathy">@caabernathy</a>), Open Source at <a href="https://github.com/facebook">Facebook</a></footer>
-          </blockquote>
-        </div>
-      </div>
-      <div class="col-sm-6 float-left pl-sm-3 pl-lg-4">
-        <div>
-          <div class="d-table-cell v-align-top pr-3">
-            <img src="https://avatars0.githubusercontent.com/u/1112?v=3&s=100" alt="" width="50" height="50" style="border-radius:25px;">
-          </div>
-          <blockquote class="d-table-cell v-align-top text-gray">
-            <p class="alt-mono-font alt-text-small">Contributing to open source projects continues to be one of the most popular 20% projects for Google engineers. Open source plays such an important role in so much of what we do, so we try to make <a href="https://opensource.google.com/docs/patching/">the process</a> as easy as possible.</p>
-            <footer class="alt-text-small">Will Norris (<a href="https://github.com/willnorris">@willnorris</a>), Open Source at <a href="https://github.com/google">Google</a></footer>
-          </blockquote>
-        </div>
-      </div>
+<div class="border-top py-5">
+  <div class="container-lg">
+    <h2 class="alt-h2 mt-6 text-center">You're in good company</h2>
+
+    <div class="pquote-container container-lg mt-3 mb-6 d-flex flex-justify-between">
+      <aside class="pquote">
+        <img src="https://avatars1.githubusercontent.com/u/691109?v=3&s=100" class="pquote-avatar" alt="avatar">
+        Open source aligns with our mission to make the world more open and connected. Working with the community gives us an opportunity to share our work and the challenges we’re aiming to solve, and encourages talented people to contribute their ideas and effort to move the world forward.
+        <p markdown="1" class="pquote-credit">
+      &mdash; Christine Abernathy (<a href="https://github.com/caabernathy">@caabernathy</a>),<br>Open Source at <a href="https://github.com/facebook">Facebook</a>
+        </p>
+      </aside>
+
+      <aside class="pquote">
+        <img src="https://avatars0.githubusercontent.com/u/1112?v=3&s=100" class="pquote-avatar" alt="avatar">
+        Contributing to open source not only allows businesses of every size to take advantage and build upon the talents of others, it’s also an avenue for those new to coding to acclimate themselves to the development process in ways that allow them to practice, experiment, and showcase themselves as valuable and contributing members of the community.
+        <p markdown="1" class="pquote-credit">
+      &mdash; Will Norris (<a href="https://github.com/willnorris">@willnorris</a>),<br>Open Source at <a href="https://github.com/google">Google</a>
```

Comment:
In the process of making this change, [@kmcrayton7](https://github.com/kmcrayton7)'s quote in `app/views/home/businesses.html.erb` got attributed to me.